src/ui/setup/index.ts

```diff
@@ -32,16 +32,15 @@ export async function bootstrapApp() {
   };
 
   const store = bootstrapStore(initialState);
+  registerStoreObserver(store, updatePrefs);
 
   if (typeof window === "undefined") return store;
 
   setupTelemetry();
-
   setupDOMHelpers();
 
   window.store = store;
 
-  registerStoreObserver(store, updatePrefs);
```

Comment:
Is it important that we moved this a couple of lines, or was it just noise from us shuffling things around? If it's just noise, maybe let's revert this change.